graphql_api/tests/test_components.py

```diff
@@ -69,7 +69,8 @@ def sample_report():
     }
 """
 
-query_repo = """
+# TODO - remove with #2290
```

Comment:
I marked the relevant existing tests as `_legacy` and a TODO for removal when I come back for cleanup in this repo. 

Then duplicated each test and made the edits needed to account for the new nested coverageAnalytics fields.